### PR TITLE
Typo fix `aggregate_pubkey` -> `bls_aggregate_pubkey`

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

File: specs/core/0_beacon-chain.md [hunk 1]
```diff
@@ -1092,8 +1092,8 @@ def verify_slashable_vote_data(state: BeaconState, vote_data: SlashableVoteData)
 
     return bls_verify_multiple(
         pubkeys=[
-            aggregate_pubkey([state.validators[i].pubkey for i in vote_data.custody_bit_0_indices]),
-            aggregate_pubkey([state.validators[i].pubkey for i in vote_data.custody_bit_1_indices]),
+            bls_aggregate_pubkey([state.validators[i].pubkey for i in vote_data.custody_bit_0_indices]),
```
Comment (hwwhww) [inline specs/core/0_beacon-chain.md]:
```suggestion
            bls_aggregate_pubkeys([state.validators[i].pubkey for i in vote_data.custody_bit_0_indices]),
```

File: specs/core/0_beacon-chain.md [hunk 2]
```diff
@@ -1092,8 +1092,8 @@ def verify_slashable_vote_data(state: BeaconState, vote_data: SlashableVoteData)
...
+            bls_aggregate_pubkey([state.validators[i].pubkey for i in vote_data.custody_bit_1_indices]),
```
Comment (hwwhww) [inline specs/core/0_beacon-chain.md]:
```suggestion
            bls_aggregate_pubkeys([state.validators[i].pubkey for i in vote_data.custody_bit_1_indices]),
```

**Comment by hwwhww** [comment]
@djrtwo Ahhh that one minute...

**Comment by djrtwo** [comment]
ohno!!

**Pull request merged**